mlflow/utils/credentials.py

```diff
@@ -90,10 +111,11 @@ def _check_databricks_auth():
         w = WorkspaceClient()
         # If credentials are invalid, `clusters.list()` will throw an error.
         w.clusters.list()
-        _logger.info(
-            "Succesfully signed in Databricks! Please run `mlflow.set_tracking_uri('databricks')` "
-            "to connect MLflow to Databricks tracking server."
-        )
+        _logger.info("Succesfully signed in Databricks!")
+        # Connect MLflow to Databricks tracking server.
+        from mlflow import set_tracking_uri
```

Comment:
can we move this import statement to the top?

Reply:
circular import at the top